HHS/Head-Start-TTADP
# [PROD]  Daily DB backup by CI & "Resource Dashboard" in left sidebar

## Description of change

[TTAHUB-2813] Daily DB backup by CI
[TTAHUB-2918] Change "Resources Dashboard" to "Resource Dashboard" in left sidebar

## How to test
TTAHUB-2813:
Look at name in left navigation bar

TTAHUB-2918:
view successful execution in ci & verify the files are in s3 under ttahub-db-backups instance
run ./bin/latest_backup.sh

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-2813
* https://jira.acf.gov/browse/TTAHUB-2918


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
